### PR TITLE
Day 4 - Menu View, Modal Image, Footer, Contact Page

### PR DESCRIPTION
**Pull request by h1tran**
- Updated cursors for hyperlinks + modal images
- Added a new link for 'About Us' in the navigation
- Updated menu view

image
- Added modal image view

image
- Updated footer links for social media

image
- Updated the contact form

image

### CODE REVIEW

File: about_us.html [hunk 1]
```diff
@@ -8,11 +8,15 @@
...
     <title>Adare Restaurant</title>
     <link rel="stylesheet" href="adare.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://kit.fontawesome.com/a076d05399.js"></script>
     <!-- Load in navigation menu as a reusable code snippet. -->
     <script>
+        $(document).ready(function() {
+            console.log("Ready!");
```
Comment (SusanZheng15) [inline about_us.html:18]: not a needed console log, usually in production code, we remove unnecessary logs

File: adare.css [hunk 2]
```diff
@@ -26,7 +26,7 @@ body {
     padding: 20px;
     font-family: Arial, Helvetica, sans-serif;
     font-size: 14px;
-    color:  white;
+    color: white;
```
Comment (SusanZheng15) [inline adare.css:29]: i feel for these types of common values that can be used in multiple places, consider having a class or a constant in which the values of the variables dont change and can be updated in one file

File: adare.css [hunk 3]
```diff
@@ -114,10 +134,16 @@ a {
...
+.a-disable {
+    text-decoration: none;
+    pointer-events: none;
+}
+
 /*Compound CSS classes*/
 nav a {
     text-decoration: none;
-    font-size: 12px;
+    font-family: 'Poppins', Helvetica, sans-serif;
```
Comment (SusanZheng15) [inline adare.css:145]: same with fonts, these can be in a separate file in which you can keep you styles in one place

File: adare.css [hunk 4]
```diff
@@ -212,5 +265,64 @@ input[type=submit] {
...
+    margin-left: 208px;
+}
+
+#lightbox {
+    /* Absolute position */
+    position: fixed;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.6);
```
Comment (SusanZheng15) [inline adare.css:277]: for this rgba value, i think having it as a variable

File: credit.html [hunk 5]
```diff
@@ -28,10 +28,52 @@
             </div>
         </div>
     </div>
-    <div class="column-4"><p></p></div>
+    <div class="column-2"><p></p></div>
     <div class="column-4">
-        <p>Credit to <b>Daniel Scott</b> for their insightful tutorial resources in the production of this website.</p>
-        <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."</p>
+        <p><b>USER LINKS</b></p>
+        <div class="row">
```
Comment (SusanZheng15) [inline credit.html:34]: white spaces o.o

**Review by SusanZheng15 — APPROVED** [review]
LGTM!

File: about_us.html (hunk 1 above)
Comment (h1tran) [reply about_us.html:18]: This has been removed, thank you for the reminder

**Pull request merged**